source/codegen/metadata/nidcpower/functions.py

```diff
@@ -1907,6 +1907,47 @@
     ],
     'returns': 'ViStatus'
   },
+  'FetchMultipleLcr': {
+    'cname': 'niDCPower_FetchMultipleLCR',
+    'parameters': [
+      {
+        'name': 'vi',
+        'direction': 'in',
+        'type': 'ViSession'
+      },
+      {
+        'name': 'channelName',
+        'direction': 'in',
+        'type': 'ViConstString'
+      },
+      {
+        'name': 'timeout',
+        'direction': 'in',
+        'type': 'ViReal64'
+      },
+      {
+        'name': 'count',
+        'direction': 'in',
+        'type': 'ViInt32'
+      },
+      {
+        'name': 'measurements',
+        'direction': 'out',
+        'size': {
+          'mechanism': 'custom-code',
```

Comment:
It looks like you can use `passed-in` instead of `custom-code` here for the mechanism. Unless I'm missing something, it looks like you just want to use the `count` field on the input message for the size here.

Reply:
I was under the impression that the APIs that use CustomTypes must be custom implemented. Am I wrong?

Reply:
APIs using CustomTypes should be able to be generated fine. I think it just happened to be the case for NiScope that all the methods using the CustomTypes were using a mechanism that wasn't supported by the codegen so they had to be custom implemented.

If you look at the fake service we used for testing, its [GetCustomTypeArray](https://github.com/doshirohan/grpc-device/blob/main/source/codegen/metadata/nifake/functions.py#L979) has an output array of a custom type using the passed-in mechanism for the size and it is generating its service handler.

Reply:
oh okay, as I mentioned in my above comment, we will not be adding these APIs as of now. If/when we take this up again, I will create a separate PR for the implementation.